front/lib/plans/stripe.ts

```diff
@@ -98,6 +98,7 @@ export const createProPlanCheckoutSession = async ({
     mode: "subscription",
     client_reference_id: owner.sId,
     customer_email: user.email,
+    payment_method_types: ["card"],
```

Comment:
```suggestion
    payment_method_types: ["apple-pay", "card", "google-pay"],
```

Reply:
(I haven't tested)

Reply:
Adding and will test on front-edge

Reply:
Stripinou said it will work

Reply:
@flvndvd it does not work, I kept card only and deployed to front edge and it still displays the apple pay option from my phone